components/org.wso2.carbon.identity.recovery/src/test/java/org/wso2/carbon/identity/recovery/password/NotificationPasswordRecoveryManagerTest.java

```diff
@@ -254,4 +268,61 @@ public void testSendRecoveryNotificationEmailTemplate(boolean isEMailOtpEnabled,
         jdbcRecoveryDataStoreMockedStatic.close();
 
     }
+
+    @Test
+    public void testUpdateNewPassword_checkPasswordValidityIsTriggered() throws Exception {
```

Comment:
Is this '_' commonly used in naming. Else you can use camel case fully

Reply:
IMO, this method naming also should be modified to indicate that is a validation of failure message from action execution